Source/items.h

```diff
@@ -165,7 +165,8 @@ enum icreateinfo_flag {
 
 enum icreateinfo_flag2 {
 	// clang-format off
-	CF_HELLFIRE = 1,
+	CF_HELLFIRE = 1 << 0,
+	CF_UIDOFFSET = (1 << 5) - 1 - CF_HELLFIRE,

```

Comment:
Perhaps this would make a bit more sense (4 bits shifted 1).
```suggestion
	CF_UIDOFFSET = ((1 << 4) - 1) << 1,
```

---

Here's another option that doesn't match our current pattern, but I thought it was interesting.
```suggestion
	CF_UIDOFFSET = ~(~0 << 4) << 1,
```